teaser/data/output/reports/model_report.py

```diff
@@ -0,0 +1,499 @@
+            prj_data[bldg_name]['UValue Groundfloor'] = sum(
+                u_values_ground_floor) / len(u_values_ground_floor)
+        else:
+            prj_data[bldg_name]['UValue Groundfloor'] = 0
+        if len(g_values_windows) > 0:
+            prj_data[bldg_name]['gValue Window'] = sum(g_values_windows) \
+                / len(g_values_windows)
+        else:
+            prj_data[bldg_name]['gValue Window'] = 0
+
+        # flat the keys
+        bldg_data = prj_data[bldg_name]
+        prj_data_flat = {}
+        for key, val in bldg_data.items():
+            if isinstance(bldg_data[key], dict):
+                for subkey in bldg_data[key].keys():
+                    prj_data_flat[str(key) + '_' + f"{subkey:03}"] = \
+                        bldg_data[key][
+                            subkey]
+            else:
+                prj_data_flat[key] = bldg_data[key]
+
+        bldg_add_list = {'OuterWall': [], 'Window': []}
+        for key in prj_data_flat.keys():
+            if key.startswith('Outerwall Area_'):
+                bldg_add_list['OuterWall'].append(key)
+            if key.startswith('Window Area_'):
+                bldg_add_list['Window'].append(key)
+        bldg_add_list['OuterWall'].sort()
+        bldg_add_list['Window'].sort()
+
+        bldg_sorted_list = [
+            'Net Ground Area',
+            'nZones'
+            'Ground Floor Area',
+            'Roof Area',
+            'Floor Height',
+            'Number of Floors',
+            'Total Air Volume',
+            *bldg_add_list['OuterWall'],
+            *bldg_add_list['Window'],
+            'Window-Wall-Ratio',
+            'Inner Wall Area',
+            'UValue Outerwall',
+            'UValue Innerwall',
+            'UValue Window',
+            'UValue Door',
+            'UValue Roof',
+            'UValue Ceiling',
+            'UValue Groundfloor',
+            'gValue Window',
+
+        ]
+        # round values
+        for key, value in prj_data_flat.items():
+            prj_data_flat[key] = round(value, 2)
+
+        bldg_data_flat_sorted = [
+            (k, prj_data_flat[k]) for k in bldg_sorted_list if
+            k in prj_data_flat.keys()]
+
+        # Draw an abstract image of the building and save it with plotly to HTML
+        interactive_fig = create_simple_3d_visualization(
+            area_north=prj_data_flat["Outerwall Area_0.0"],
+            area_east=prj_data_flat["Outerwall Area_90.0"],
+            area_south=prj_data_flat["Outerwall Area_180.0"],
+            area_west=prj_data_flat["Outerwall Area_270.0"],
+            height=prj_data_flat["Floor Height"],
+            window_area_north=prj_data_flat["Window Area_0.0"],
+            window_area_east=prj_data_flat["Window Area_90.0"],
+            window_area_south=prj_data_flat["Window Area_180.0"],
+            window_area_west=prj_data_flat["Window Area_270.0"],
+            num_floors=prj_data_flat["Number of Floors"],
+            roof_angle=30)
```

Comment:
hard coded roof_angle. Will this lead to problems in future?

Reply:
roofs are on #todo, this is also noted in the code.